san-francisco.md

```diff
@@ -118,7 +117,6 @@ Try [Handy](https://www.handy.com/) for house cleaning.
 * Hoodline
   [Twitter account](https://twitter.com/hoodlinesf) and
   [iPhone app](https://itunes.apple.com/us/app/hoodline/id814699998?mt=8)
-* [The Bold Italic](http://www.thebolditalic.com/articles)
```

Comment:
It looks like The Bold Italic is still running articles: https://thebolditalic.com/

How about changing this to remove the `/articles` instead of deleting it?